Correction

yusei234 opened the pull request:
corrections of various bugs

Commit:
Corrections of some bugs

yusei234 commented:
On my linux computer there is a bug whith the fool assets, but not on my windows one. So the executable seems all good.
It would be good if you both try the games files to see if it works like it should (try going till the last round to see everything please).

pedro-dietrich commented on scripts/cards/major_arcanas/the_fool.gd:
```diff
@@ -9,6 +9,7 @@ func _init() -> void:
 
 
 func get_major_images_path() -> String:
+	print("major_image_path of the fool called")
 	return "res://assets/card/major_arcanas/the_fool/"
```
This print is only for debug. Remove it.

pedro-dietrich commented on scripts/utils/level.gd:
```diff
@@ -53,9 +53,12 @@ func increment() -> void:
 
 func get_card_points(played_card: ElementalCard, played_cards, bonus_arcanas):
 	# Change the value of the card depending on the major arcana rule
+	print("Before malus effect:", played_card.point)
 	played_card.point = get_malus_arcana().malus_effect_on_points(played_cards, LVL_MAX_CARDS_PLAYED[level])
+	print("after malus effect:", played_card.point)
 	for major_bonus in bonus_arcanas:
 		played_card.point = major_bonus.major_arcana.bonus_effect_on_points(played_cards, LVL_MAX_CARDS_PLAYED[level])
+		print("after bonus effect of ", major_bonus.id, " :", played_card.point)
 	
```
More debug prints to remove.

pedro-dietrich commented on scripts/cards/major_arcanas/the_lovers.gd:
```diff
@@ -30,9 +30,9 @@ func malus_effect_on_points(active_cards: Array[ElementalCard], _max_active_card
+	var base_points: float = target_card.point
 
-	if(active_cards.size() > 1 and typeof(target_card.element) != typeof(active_cards[-2].element)):
+	if(active_cards.size() > 1 and target_card.element.get_label_text() != active_cards[-2].element.get_label_text()):
```
Why compare the label texts? This does not seem any better than comparing the types.

yusei234 replied:
because the type that is compare is always element, so it is always the same with typeof

pedro-dietrich commented on scripts/game_logic.gd:
```diff
@@ -179,7 +178,9 @@ func _on_card_played(card_id: int) -> void:
 
 func next_malus() -> void:
-	if (level.is_last_level()): level.malus_arcana_card = card_factory.fool_arcana_card()
+	if (level.is_last_level()): 
+		list_major_arcana = range(0, 1)
+		level.malus_arcana_card = card_factory.random_major_arcana_card(list_major_arcana)
```
Why this change? It was only for testing, right?

pedro-dietrich replied:
Then revert it back before merging.

pedro-dietrich reviewed (commented):
GitHub has a special way to handle the binaries (executables).

Instead of placing it on the repo normally, use the `Releases` feature.

I would also like to know what is the purpose of the `Tarot_Journey.sh` and what are the `.x86_64` and `.console.exe` files...

yusei234 committed:
PR reviews

yusei234 committed:
Del debug changes

pedro-dietrich committed:
Reduce The Fool texture atlases

The previous textures for The Fool foreground and middleground were too
big, causing crashes on certain cases.
